### DISCUSSION TITLE
how to access an external link on OneDrive?

### DISCUSSION THREAD

**Discussion by parkus83** [O365/python-o365#1020] [Q&A]
Hi all, I am trying to access the folder linked to an external link as shown in the figure. I have read and write permissions on this folder, but it seems that the drive.get_items() method is unable to see it. 
Btw with drive.get_items() I can see all the other "standard" folder and files enlisted in this figure.
Is there a way to access this external link through O365?

![Screenshot 2023-10-12 alle 13 08 56](https://github.com/O365/python-o365/assets/122524996/16deaead-eeea-4775-8256-791e70e73d61)

Regards

**Comment by alejcas** [comment]
There's this method:  `Drive.get_shared_with_me`
